firestore/src/include/firebase/firestore/map_field_value.h

```diff
@@ -36,7 +36,11 @@ using MapFieldPathValue = std::unordered_map<FieldPath, FieldValue>;
 // module. A workaround for deserialization cross reference compiler
 // crashes https://github.com/apple/swift/issues/70253
 using FieldValueVector = std::vector<FieldValue>;
+#if defined(_WIN32)
 using FieldValueVectorConstIterator = std::_Vector_const_iterator<std::_Vector_val<std::_Simple_types<FieldValue>>>;
```

Comment:
Any chance Windows would now work with `std::vector<FieldValue>::const_iterator` too?

Reply:
Unlikely, however @hyp is working on fixing the compiler. We should hopefully be able to remove this soon, but in the mean time, this change should allow me to continue to make progress for Android.